lib/chef/data_bag.rb

```diff
@@ -70,11 +70,11 @@ def to_json(*a)
     end
 
     def chef_server_rest
-      @chef_server_rest ||= Chef::REST.new(Chef::Config[:chef_server_url])
+      @chef_server_rest ||= Chef::ServerAPI.new(Chef::Config[:chef_server_url])
```

Comment:
i was thinking about going through and require'ing server_api explicitly, but we don't actually require rest anywhere right now. So yes, I think it'll make you sad, and I'll do the explicit require and make you less sad.